content/docs/concepts/remote_write_spec_2_0.md

```diff
@@ -0,0 +1,394 @@
+
+The remote write protocol contains opportunities for batching, e.g. sending multiple samples for different series in a single request. It is not expected that multiple samples for the same series will be commonly sent in the same request, although there is support for this in the protocol.
+
+<!---
+TODO(bwplotka): Challenge last sentence (e.g. OTel coll usage)?
+-->
+The remote write protocol is not intended for use by applications to push metrics to Prometheus remote-write-compatible Receiver. It is intended that a Prometheus remote-write-compatible sender scrapes instrumented applications or exporters and sends remote write messages to a server.
+
+<!---
+TODO(bwplotka): Add support for 2.0 to those suites 
+-->
+A test suite can be found at https://github.com/prometheus/compliance/tree/main/remote_write_sender.
+
+### Glossary
+
+For the purposes of this document the following definitions MUST be followed:
+
+* a "Sender" is something that sends Prometheus Remote Write data.
+* a "Receiver" is something that receives Prometheus Remote Write data.
+* a "Sample" is a pair of (timestamp, value).
+* a "Histogram" is a pair of (timestamp, histogram value).
+* a "Label" is a pair of (key, value).
+* a "Series" is a list of samples, identified by a unique set of labels.
+
+## Definitions
+
+### Protocol
+
+The Remote Write Protocol MUST consist of RPCs with the request body encoded using a Google Protobuf 3 message. The protobuf encoding MUST use either of the following schemas: 
+
+<!---
+TODO(bwplotka): Do we deprecate it?
+-->
+* [`prometheus.WriteRequest`](./remote_write_spec.md#protocol) introduced in the Remote Write 1.0 specification. As of 2.0 the `prometheus.WriteRequest` message is deprecated.
+* `io.prometheus.write.v2.Request` introduced in this specification and defined [below](#ioprometheuswritev2request-proto-schema). Senders SHOULD use `io.prometheus.write.v2.Request` when possible.
+
+Sender MUST send encoded and compresses proto message in the body of an HTTP POST request and send it to the Receiver via HTTP at a provided URL path. The Receiver MAY specify any HTTP URL path to receive metrics.
+
+Sender MUST send the following "reserved" headers with the HTTP request:
+
+* `Content-Encoding: <compression>`
+
+  Content encoding request header MUST follow [the RFC 9110](https://www.rfc-editor.org/rfc/rfc9110.html#name-content-encoding). Sender MUST use the `snappy` value. More compression algorithms might come in 2.x or beyond.
+
+* `Content-Type: application/x-protobuf` or `Content-Type: application/x-protobuf;proto=<fully qualified name>`
+  
+  <!---
+  TODO(bwplotka): The framing here is a bit inconsistent with deprecation policy. We can either mention sender MUST use either 1.0 or 2.0 message. Or we can say sender MUST use 2.0 message and it MAY/SHOULD use 1.0 message against 1.x receiver. What do we prefer?
+  -->
```

Comment:
`MUST try the 2.0 format, and MAY fall back to using the 1.0 format if the receiver rejects 2.0 messages because it does not yet have support for the 2.0 format`?